test/parallel/test-worker-eventlooputil.js

```diff
@@ -34,9 +34,10 @@ function workerOnMetricsMsg(msg) {
   }
 
   if (msg.cmd === 'spin') {
+    const elu = eventLoopUtilization();
     const t = now();
     while (now() - t < msg.dur);
-    return this.postMessage(eventLoopUtilization());
+    return this.postMessage(eventLoopUtilization(elu));
```

Comment:
just my opinion, that the caller (main thread) making calculations / differences etc. rather than the callee doing that. but did it matter in any way how the test failed?

Reply:
The API to be tested in this test is `worker.performance.eventLoopUtilization()` which is intended to be called from main thread.
The test uses also the similar API `perf_hooks#performance.eventLoopUtilization()` which gives the times for the current thread and compares the results.
Maybe the test could be simplified to avoid this mix but my intention here is only to get the test more stable.